language-server/src/main/java/org/sonarlint/languageserver/ServerInfoCache.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * SonarLint Language Server
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.languageserver;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import javax.annotation.Nullable;
+
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+/**
+ * Cache of server connection details. Used for updating server and module storage.
+ */
+class ServerInfoCache {
+
+  private final Logger logger;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". ([squid:S1312](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&pullRequest=192&issues=AWND8neu6NykrOWnW1BZ&open=AWND8neu6NykrOWnW1BZ)

[ISSUE_KEY]: AWND8neu6NykrOWnW1BZ